### ISSUE TITLE
"<, <=, >=, >" operators not supported on tuples

### ISSUE THREAD

**Issue by rchl** [microsoft/pyright#1170]
**Describe the bug**
This code triggers a type error:

```py
(1, 2, 3) > (1, 2, 2)
```

```
Operator ">" not supported for types "Tuple[Literal[1], Literal[2], Literal[3]]" and "Tuple[Literal[1], Literal[2], Literal[2]]"
             	            	          	  Operator ">" not supported for types "Tuple[Literal[1], Literal[2], Literal[3]]" and "Tuple[Literal[1], Literal[2], Literal[2]]"
```

**To Reproduce**
Type the code in the file.

**Expected behavior**
Should raise no issues as Python supports that.

**Screenshots or Code**
![Screenshot 2020-11-13 at 22 24 06](https://user-images.githubusercontent.com/153197/99122782-46399880-25ff-11eb-8ae9-196ba5341b99.png)

**VS Code extension or command-line**
Running in a ST client.

**Additional context**
It works when using `==` operator.

**Comment by erictraut** [comment]
Yes, this is the expected behavior. It's due to the way types are inferred for tuple expressions. See [here](https://github.com/microsoft/pyright/blob/master/docs/type-inference.md#tuple-expressions) for details.

**Comment by rchl** [comment]
Expected due to current limitations or just not expected?

Is there a way to "coerce" types in this case to something more generic that works? This doesn't at least:

```py
t1: Tuple[int, ...] = (1, 2, 3)
t2: Tuple[int, ...] = (1, 2, 4)

t1 >= t2
```

**Comment by erictraut** [comment]
The correct workaround is the one you provided above. The fact that this doesn't work is a bug. It's a regression that was introduced about a month ago when I added support for subscripted `tuple` expressions, which are now supported in Python 3.9, in addition to subscripted `Tuple`. I've fixed this bug and added it to our test suite to prevent similar future regressions.

**Comment by erictraut** [comment]
This is now addressed in 1.1.86, which I just published. It will also be included in the next release of pylance.
